runtime/Cpp/cmake/antlr4-runtime.cmake.in

```diff
@@ -8,3 +8,5 @@ set_and_check(ANTLR4_LIB_DIR "@PACKAGE_ANTLR4_LIB_DIR@")
 include(${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake)
 
 check_required_components(antlr)
+
+find_dependency(Threads)
```

Comment:
I think this is also missing an include:
```suggestion
include(CMakeFindDependencyMacro)
find_dependency(Threads)
```

Reply:
This might also need to go before the `include(${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake)` line, since the exported targets will reference the Threads dependency.